.github/workflows/build-release.yml

```diff
@@ -0,0 +1,73 @@
+name: build-release
+
+on:
```

Comment:
I believe the triggers for this workflow were changed for testing purposes. However, since this is the new workflow to build a new release, this should be changed to trigger only on pushing a new version-tag, like:
```
on:
  push:
    tags:
      - 'v*' # only release a versioned tag, such as v.X.Y.Z
```

Reply:
No I think this is correct based on what we have already discussed. This action will run on all pull requests (and syncs) to master BUT the publish and deploy steps will only run if v* tag is present.

Reply:
Ok, yeah - that makes sense. Thanks for reminding me.

Reply:
Sorry @R7L208 - my bad, we should roll this back to the way it was. Also take note that we just merged a change from @nfx that renamed the build workflow file - so let's be careful on getting this merged without conflicts